cdap-ui/app/cdap/services/global-events.js

```diff
@@ -22,11 +22,13 @@ export default {
   APPUPLOAD: 'APPICATION_UPLOAD',
   ARTIFACTUPLOAD: 'ARTIFACT_UPLOAD',
   CLOSEMARKET: 'CLOSE_MARKET',
+  CLOSEPLUSBUTTON: 'CLOSE_PLUS_BUTTON',
```

Comment:
Could we make this event name a bit more descriptive? Maybe something like "CLOSE(or OPEN)WIZARD"? Right now it sounds like the button itself is closing, and doesn't quite follow the pattern of the other global events.

Reply:
Have renamed it to OPEN/CLOSE RESOURCECENTER to mimic OPEN/CLOSE MARKET.